src/wp-includes/block-bindings.php

```diff
@@ -66,3 +66,15 @@ function unregister_block_bindings_source( $source_name ) {
 function get_all_registered_block_bindings_sources() {
 	return WP_Block_Bindings_Registry::get_instance()->get_all_registered();
 }
+
+/**
+ * Retrieves a registered block bindings source.
+ *
+ * @since 6.5.0
+ *
+ * @param string $source_name The name of the source.
+ * @return array|null The registered block bindings source, or `null` if it is not registered.
+ */
+function get_registered_block_bindings_source( string $source_name ) {
```

Comment:
Should this be `get_block_bindings_source`? It seems a bit simpler while self-explanatory as well.

Reply:
Like it 👍

Reply:
Done with https://github.com/WordPress/wordpress-develop/pull/6016/commits/b337f0112f1d5bc7ae4325d67d8bad488300d5a7.